### ISSUE TITLE
Elementor js errors

### ISSUE THREAD

**Issue by sukhsingh32** [elementor/elementor#14571]
Can someone help me to resolve this issue.
common.min.js?ver=3.1.3:2 Uncaught ReferenceError: originalEvent is not defined
    at onError (common.min.js?ver=3.1.3:2)
    at dispatch (jquery.min.js?ver=3.5.1:2)
    at v.handle (jquery.min.js?ver=3.5.1:2)

**Comment by LilMinmax** [comment]
We have the same issue with Elementor 3.2.3 and Elementor Pro 3.2.2. The editor refuses to load.

Uncaught ReferenceError: originalEvent is not defined

This happens in Firefox on PC, but not Edge or Chrome on PC. It happens on all browsers on the Mac.

This happens when you view the page as well, not just in the editor, but only when you're logged in to wordpress.

We get another error when loading the editor:

Uncaught DOMException: Permission denied to access property "elementorFrontend" on cross-origin object

Nothing seems to be cross-origin, and we have the X-Frame-Options set to sameorigin.

We rolled back to 3.0.9 for each and still same problem. We disabled all other plugins and same problem. We removed GTM from our site so no additional code was running and same issue. We used the troubleshooter and same with just the two plugins on.

Our editors on Macs can't do any work.

Let me know if you need any additional information.

**Comment by blackashconsulting** [comment]
Also having this issue and it's happening on every site I operate, which is...a lot. It eventually ends up in a 524 error.

**Comment by LilMinmax** [comment]
@sukhsingh32 I feel like this issue should be retitle to indicate that the editor doesn't work at all, the current subject doesn't scan as urgent or that problematic. Can you add that to the title? If not, I'll create a new issue

**Comment by blackashconsulting** [comment]
> @sukhsingh32 I feel like this issue should be retitle to indicate that the editor doesn't work at all, the current subject doesn't scan as urgent or that problematic. Can you add that to the title? If not, I'll create a new issue

Please create a new issue- I've created 2 at this point, the first was brushed off as a Cloudflare problem when it effects sites that don't use Cloudflare as well. I can't edit any of my sites, and nor can any of my clients.

**Comment by LilMinmax** [comment]
@blackashconsulting if you want to add details:

https://github.com/elementor/elementor/issues/14928

**Comment by DorShahar** [comment]
Thanks for reporting,
Please follow guidelines for opening a new [issue report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md), so we can help you better.
This report is missing key information: 
* Prerequisites.
* **Steps to replicate.**
* Isolating the problem.
* **System info.**
* One issue per thread

Without it, we simply can't help.

Thanks.

**Comment by philcarbo** [comment]
@DorShahar check issue #15121

**Comment by advokatb** [comment]
Will this issue be fixed? It's very annoying.

**Comment by nitingohel** [comment]
I am also getting soo many error while install fresh WP with elementor in my localhost.

![image](https://user-images.githubusercontent.com/5137595/147473602-13284058-834a-4696-b1c6-f50bfd91dad8.png)

Anyone please help 🙏

**Comment by steve231293** [comment]
hi @udidol

Could you please fix this bug? This bug make editor can't load.

![Screenshot at Jan 10 15-36-07](https://user-images.githubusercontent.com/7134267/148737584-eb5c5241-6672-4c03-85a3-e4dd8e0698f7.png)

For anyone to quick fix you can edit common.js 

Change 
```
self.addError({
      type: error.name,
      message: error.message,
      url: originalEvent.filename,
      line: originalEvent.lineno,
      column: originalEvent.colno
    });
```

```
self.addError({
      type: error.name,
      message: error.message,
      url: event.originalEvent.filename,
      line: event.originalEvent.lineno,
      column: event.originalEvent.colno
    });
```

**Comment by udidol** [comment]
A fix for this bug will be released in one of the next couple upcoming releases.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.5.4

Feel free to update

Thanks!
